cpp/pipelines/add_authority_beacon_information.cc

```diff
@@ -21,15 +21,65 @@
 #include <cstdlib>
 #include "BeaconFile.h"
 #include "MARC.h"
+#include "FileUtil.h"
 #include "StringUtil.h"
 #include "util.h"
+#include <iostream>
 
 
 namespace {
 
 
+class TypeFile {
+
```

Comment:
We don't use blank lines for nested classes and structs.

Reply:
Done